ES/ESmy023.xml

```diff
@@ -276,8 +276,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                            <material key="parchment"/>
                         </support>
                         <extent><measure unit="leaf" quantity="91">91</measure>
-                              <measure unit="leaf" type="blank"/>
-                              <locus target="#1rv #2rv #3rv #91rv"/>
+                              <measure unit="page" type="blank"/>
```

Comment:
do we use page here? I thought we count in pages only for paginated mss

Reply:
We do; I changed back, but there was a consideration behind, maybe to discuss separately.